src/NATS.Client.JetStream/NatsJSContext.Streams.cs

```diff
@@ -76,6 +76,27 @@ public partial class NatsJSContext
         return response;
     }
 
+    /// <summary>
+    /// Deletes a message from a stream.
+    /// </summary>
+    /// <param name="stream">Stream name to delete message from.</param>
+    /// <param name="request">Delete message request.</param>
+    /// <param name="cancellationToken">A <see cref="CancellationToken"/> used to cancel the API call.</param>
+    /// <returns>Delete message response</returns>
+    /// <exception cref="NatsJSException">There was an issue retrieving the response.</exception>
+    /// <exception cref="NatsJSApiException">Server responded with an error.</exception>
+    public async ValueTask<StreamMsgDeleteResponse> DeleteMessageAsync(
```

Comment:
Should we implement a convenience method in the stream class as well?

Reply:
Good point. Done!